spec/rails_helper.rb

```diff
@@ -189,3 +189,14 @@ def initialize(meta)
   config.cache_dir = Rails.root.join("tmp", "cache", "carrierwave#{ENV["TEST_ENV_NUMBER"]}")
   config.enable_processing = false
 end
+
+# Create external registries
+RSpec.configure do |config|
+  config.before(:all) do
+    ExternalRegistry.create_all
```

Comment:
Do we really need to run this before every test?

Reply:
This is only invoked once before / after an entire test run, but I can move it into the test suites that depend on it—it's a little more explicit. Updated in 8361b50b